data/brands/shop/convenience.json

```diff
@@ -5798,6 +5798,18 @@
         "name:zh": "超市发",
         "shop": "convenience"
       }
+    },
+    {
+      "displayName": "十足",
+      "id": "shizu-zhejiang-cn",
+      "locationSet": {"include": ["cn"]},
+      "tags": {
+        "brand": "十足",
+        "brand:en": "Shizu",
+        "name": "十足",
+        "name:en": "Shizu",
+        "shop": "convenience"
```

Comment:
You need to have a wikidata item linked to this entry so that it can be suggested in editor

See: https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required

Reply:
Hmm what if there is no wikipedia and Wikidata item for it? It's not on either of those.

Reply:
> Hmm what if there is no wikipedia and Wikidata item for it? It's not on either of those.

Create one if needed (This require a wikimedia account)

![图片](https://user-images.githubusercontent.com/42690037/218300746-8b51de3e-db33-485e-b7f3-1149969a5e7a.png)

You can add more information on it: https://www.wikidata.org/wiki/Q116763971